financial-connections/src/main/java/com/stripe/android/financialconnections/model/serializer/EntrySerializer.kt

```diff
@@ -0,0 +1,26 @@
+package com.stripe.android.financialconnections.model.serializer
```

Comment:
are these serializer FC specific? otherwise could we move them to stripe-core like [this one](https://github.com/stripe/stripe-android/blob/4a5d57e6a15d8845a12d668c47afcfaf5c5022b4/stripe-core/src/main/java/com/stripe/android/core/model/serializers/CountryListSerializer.kt#L19)

Reply:
They're specific to FC's server driven UI.